plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,632 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024, IamLunchbox <r.grieger@hotmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: proxmox_backup
+author: "Raphael Grieger (@IamLunchbox) <r.grieger@hotmail.com>"
+short_description: Start a VM backup in Proxmox VE cluster
+version_added: 10.0.0
```

Comment:
10.0.0 is already out, so you cannot add new content to it :)
```suggestion
version_added: 10.1.0
```

Reply:
time flies...